styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  Box-sizing: border-box; }
+
+a{
+   text-decoration: none;
+}
+a:link {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+}
```

Comment:
You should be able to eliminate these two rules, as they will inherit from the rule on line 5 👍🏻